### PR TITLE
newsflash: 1.1.1 -> 1.2.2

### PR DESCRIPTION
**Pull request by kira-bruneau**
###### Motivation for this change
Upgrade to the latest version: https://gitlab.com/news-flash/news_flash_gtk/-/tags/1.2.2

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by MetaDark** [comment]
Result of `nixpkgs-review pr 111336` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>newsflash</li>
  </ul>
</details>

File: pkgs/applications/networking/feedreaders/newsflash/default.nix [hunk 1]
```diff
@@ -19,23 +19,22 @@
...
-  cargoSha256 = "0rnrdh9ganj63hf9j890yj9pahcgza95z7x020w72mbb4648hq26";
+  cargoSha256 = "0a33k8wxvhxh1gc5xacw00hnvaxmijkv0x92wj5f5dqp0fh4da4j";
 
   patches = [
     ./no-post-install.patch
   ];
 
   postPatch = ''
-    chmod +x build-aux/cargo.sh
     patchShebangs .
```
Comment (Stunkymonkey) [inline pkgs/applications/networking/feedreaders/newsflash/default.nix:38]: it is correct, that the `chmod` is no longer needed
have you tested if removing the `patchShebangs` does work?
consider: https://gitlab.com/news-flash/news_flash_gtk/-/merge_requests/40

Comment (MetaDark) [reply pkgs/applications/networking/feedreaders/newsflash/default.nix:38]: Yep, I tried removing `patchShebangs`, and the build failed. It seems like meson's `find_program` doesn't find the interpreter when using `/usr/bin/env bash`.

Comment (MetaDark) [reply pkgs/applications/networking/feedreaders/newsflash/default.nix:38]: It looks like `/usr/bin/env` isn't available in the nix sandbox: https://github.com/NixOS/nix/issues/4031.

Comment (Stunkymonkey) [reply pkgs/applications/networking/feedreaders/newsflash/default.nix:38]: thanks. something i did not know :+1:

**Comment by MetaDark** [comment]
Result of `nixpkgs-review pr 111336` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>newsflash</li>
  </ul>
</details>

**Comment by MetaDark** [comment]
Result of `nixpkgs-review pr 111336` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>newsflash</li>
  </ul>
</details>

**Pull request merged**